NFIQ2/NFIQ2Algorithm/include/nfiq2/features/FJFXMinutiaeQualityFeatures.h

```diff
@@ -32,15 +32,9 @@
 /* Ideal Mean of pixel values in a neighborhood. */
 #define IDEALMEAN 127
 
-#ifndef WITHOUT_BIOMDI_SUPPORT
 #if defined WINDOWS || defined WIN32
 #include <sys/queue.h>
 #endif
```

Comment:
I think this can be removed too.

Reply:
Yes, It actually does not compile on Windows if it's left in.